### ISSUE TITLE
More than one server

### ISSUE THREAD

**Issue by KrustyGamer134** [xamayca/ASCENDEDSERVER#2]
Would it be possible to have a way to run more than once server at a time? 

Right now, if you have one server open and run the start.bat to start a second map it sees that there is one server running and closes it.

Not sure if this is something that can be done with the server's name or something to check and see if it is currently running.

**Comment by xamayca** [comment]
Yes we working on adding cluster id, fix update check, host a second a many other's server with the same script, and some fix soon. Thanks !
